docs/cli-reference/clawback.md

```diff
@@ -126,71 +59,152 @@ Options:
 | -h            | --help            | None    | False    | Show a help message and exit                                                                             |
 
 <details>
-<summary>Example 1</summary>
+<summary>Example</summary>
 
-Request:
+First, the Sender creates a new clawback coin with a 60-second timelock:
 
 ```bash
+clawback create -t txch1czxl4c42kwem8jllm4xp78jt7t0j0pu9uh9xw5jddegcu703xnpsqaaxxx -a 0.1 -l 60 -m 0.000275
+```
+
+Result:
 
+```bash
+Created Coin with ID: ef4b69e65e99261d6e30c8d2d331a8ed84995f3452b95aaa944f76a0f9af74c5
+Coin { parent_coin_info: f5e408a15299df27a00d0403ea50ed29a06f87a31b638cff1b63ef3e6060b45a, puzzle_hash: 2e91566d9549cb909109d147c79b457d15ced2d8a24dea5a125b1e5c05cdd1ec, amount: 100000000000 }
 ```
 
-Response:
+The Sender can run the `clawback show` command to show the details of each of their clawback coins:
 
+```bash
+clawback show
 ```
 
+Result:
+
+```bash
+Updating coin records...
+
+
+Coin ID: ef4b69e65e99261d6e30c8d2d331a8ed84995f3452b95aaa944f76a0f9af74c5
+Amount: 0.1 XCH (100000000000 mojos)
+Timelock: 60 seconds
+Time left: 23 seconds
 ```
 
-</details>
+The timelock expires when the `Time left:` value reaches `0 seconds`:
 
-<details>
-<summary>Example 2</summary>
+```bash
+clawback show
+```
 
-Request:
+Result:
 
 ```bash
+Updating coin records...
+
 
+Coin ID: ef4b69e65e99261d6e30c8d2d331a8ed84995f3452b95aaa944f76a0f9af74c5
+Amount: 0.1 XCH (100000000000 mojos)
+Timelock: 60 seconds
+Time left: 0 seconds
 ```
 
-Response:
+For your reference, the Recipient wallet currently has a balance of 0 XCH:
 
+```bash
+chia wallet show -f 1231588123
 ```
 
+Result:
+
+```bash
+Wallet height: 2405060
+Sync status: Synced
+Balances, fingerprint: 1231588123
+
+Chia Wallet:
+   -Total Balance:         0.0 txch (0 mojo)
+   -Pending Total Balance: 0.0 txch (0 mojo)
+   -Spendable:             0.0 txch (0 mojo)
+   -Type:                  STANDARD_WALLET
+   -Wallet ID:             1
 ```
 
-</details>
+The Recipient (or anyone else) can view the status of the clawback coin if they know the coin ID:
 
-<details>
-<summary>Example 3</summary>
+```bash
+clawback show -c ef4b69e65e99261d6e30c8d2d331a8ed84995f3452b95aaa944f76a0f9af74c5
+```
 
-Request:
+Result:
 
 ```bash
+Updating coin records...
+
 
+Coin ID: ef4b69e65e99261d6e30c8d2d331a8ed84995f3452b95aaa944f76a0f9af74c5
+Amount: 0.1 XCH (100000000000 mojos)
+Timelock: 60 seconds
+Time left: 0 seconds
 ```
 
-Response:
+The value of `Time left:` is `0 seconds`, which indicates that the Recipient can now run the `clawback claim` command to claim the coin:
```

Comment:
There is a slight edge case where the timer might have expired, but a block still hasn't been farmed with a timestamp after the expiry. So in that case you'd get a "spending too early" error, but would be able to submit once the next block is farmed. Over to you whether we need to articulate that though